multicluster/apis/multicluster/v1alpha1/tunnelendpointimport_types.go

```diff
@@ -0,0 +1,78 @@
+/*
+Copyright 2022 Antrea Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+import (
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+type TunnelEndpointImportConditionType string
+
+const (
+	// TunnelEndpointImportReady indicates TunnelEndpoint is imported successfully.
+	TunnelEndpointImportReady TunnelEndpointImportConditionType = "Ready"
```

Comment:
Do we need to define a new type for each resource type? Or a single ResourceImportReady to be shared by all?

Reply:
It should be able to share. I will check and refine.